spec/Game_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rspec'
+require'./lib/Game'
+require "./lib/Player"
+require"./lib/Board"
+
+describe Game do 
+   describe '#winner?' do
+
+      let(:board) { Board.new}
+      let(:player_1) {Player.new("Player 1", "X", board)}
+      let(:player_2) {Player.new("Player 2", "O", board)}
+      let(:game) {Game.new(board, player_1, player_2)}
```

Comment:
No need to use it twice, needs to put it under `describe game` block and remove the other within `describe #play` for `let`